src/schema/objects/entities.yaml

```
@@ -286,6 +276,15 @@ sample:
     The label MUST be unique per subject and is RECOMMENDED to be unique throughout the dataset.
   type: string
   format: label
+seg:
+  name: seg
+  display_name: segmentation
+  description: |
+    The `seg-<label>` key/value pair corresponds to a custom label the user
+    MAY use to distinguish different segmentations.
+    This entity is only applicable to derivative data.
```

The double-newline ensures that a newline is rendered.

```suggestion
    MAY use to distinguish different segmentations.

    This entity is only applicable to derivative data.
```

---

When I looked through not all entities had this, but thanks for fixing that.